src/create.cpp

```diff
@@ -815,6 +849,46 @@ namespace create {
       return 0;
     }
   }
+  
+  	  uint16_t Create::getCliffSignalLeft() const {
+	  if (data->isValidPacketID(ID_CLIFF_LEFT_SIGNAL)) {
```

Comment:
Have you tested this functionality? I don't expect `ID_CLIFF_LEFT_SIGNAL` to be evaluated as a valid packet since it is not added as a packet to request for any protocol in [data.cpp:13](https://github.com/AutonomyLab/libcreate/blob/master/src/data.cpp#L13). You can add them to that constructor for protocols `V_2 | V_3`, but beware the warning regarding flooding the serial. After you've made the changes you should be able to check if there are problems on the serial with calls to `Create::getNumCorruptPackets()`.